Test commit for Benzo aka asshoe Sensei

dvu1019 opened the pull request:
Display value of ID and name for first issue in array

dvu1019 committed:
Test commit for Benzo aka asshoe Sensei
Display value of ID and name for first issue in array

mostlyerror commented:
Cool. Now leave it open/unmerged while you build up your commits on the PR. Then we can take a look at all of the chunks in entirety.

dvu1019 committed:
Added the insertAdjacentHtml method that inserts a text as html in a specified position.
Used afterend to append the ticket details to the h2 element of issue list.

dvu1019 committed:
Update Issues list to automatically load the list on load

dvu1019 committed:
Removed buttons to add and list issues
List of issues should automatically load on hitting page
adding issues will append to the list
